src/components/Menu/Menu.stories.tsx

```diff
@@ -164,6 +180,48 @@ export const WithCustomButton: Story = {
   },
 };
 
+/**
+ * When using `.PlainButton`, you can also use a `Button` instance within it to specify any other
+ * button details, including alternative icon/text combinations, ranks, etc.
+ *
+ * **Note**: here we use `React.Fragment` to avoid nesting `<button>` tags within a `<button>` tag.
+ */
+export const WithCustomSecondaryButton: Story = {
```

Comment:
`DataTable` actions need this functionality, so opened this PR to test if we needed changes to menu. Turns out, we do not :)